starter-core/src/test/groovy/io/micronaut/starter/feature/other/HttpSessionSpec.groovy

```diff
@@ -75,7 +75,7 @@ class HttpSessionSpec extends ApplicationContextSpec implements CommandOutputFix
         then:
         template.contains("""
     <dependency>
-      <groupId>io.micronaut</groupId>
+      <groupId>io.micronaut.session</groupId>
```

Comment:
Use BuildVerifier API to test dependencies

Reply:
@Sergio that API is in the 3.9.x branch and needs to be merged up first. I mentioned that on the discovery-core PR as well.